packages/utils.js

```diff
@@ -117,9 +109,21 @@ export function throttle (callback, ms) {
   }
 }
 
+/**
+ * toggleAttribute (Ponyfill for IE and Edge, fixes #299)
+ * @param {Element} el  The new throttled function
+ * @param {String} name The name of the attribute to be toggled
+ * @param {Boolean} force Determine whether the attribute should be added or removed
```

Comment:
Jeg skjønner ikke hva `force` gjør

Reply:
Godt poeng, var for å være i tråd med https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute :)

Reply:
Will continue using `force` as `value` is misleading. Changed description and added link to original docs: https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute.
Also see `force` used as parameter name in `.toggle` here https://developer.mozilla.org/en-US/docs/Web/API/Element/classList :)